File: src/main/java/org/example/model/Member.java
package org.example.model;

import jakarta.persistence.*;

import java.time.LocalDate;

@Entity
@Table(name = "Members")
public class Member {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long memberId;

    @Column(nullable = false)
    private String fullName;

    private LocalDate membershipDate;

    // Getters and Setters
    public Long getMemberId() {
        return memberId;
    }

    public void setMemberId(Long memberId) {
        this.memberId = memberId;
    }

    public String getFullName() {
        return fullName;
    }

    public void setFullName(String fullName) {
        this.fullName = fullName;
    }

    public LocalDate getMembershipDate() {
        return membershipDate;
    }

    public void setMembershipDate(LocalDate membershipDate) {
        this.membershipDate = membershipDate;
    }
}

File: src/app/models/member.ts
export class Member {
    memberId: number;
    fullName: string;
    membershipDate: Date;
  
    constructor(memberId: number, fullName: string, membershipDate: Date) {
      this.memberId = memberId;
      this.fullName = fullName;
      this.membershipDate = membershipDate;
    }
  
  }
  